constants/androidWebkitVersions.js

```diff
@@ -18,4 +18,6 @@ module.exports = {
   '4.2': '534.30',
   '4.3': '534.30',
   '4.4': '537.36',
+  '5.0': '537.36 (Chrome 37)',
+  '5.1': '?'
```

Comment:
Why the `?` here?

Reply:
> Why the `?` here?

This was a placeholder for me to find the SDK version - but, couldn't find it. We can remove this line if that's confusing.

Reply:
Yeah, IMO, better to remove it if unknown.

Reply:
Removed.